adapter/internal/api/apis_process.go

```diff
@@ -144,10 +146,50 @@ func processFileInsideProject(apiProject *model.ProjectAPI, fileContent []byte,
 				// TODO: (VirajSalaka) Create standard error handling mechanism
 				return errors.New("certificate Validation Error")
 			}
-
 			if fileNameArray := strings.Split(fileName, string(os.PathSeparator)); len(fileNameArray) > 0 {
 				certFileName := fileNameArray[len(fileNameArray)-1]
 				apiProject.UpstreamCerts[certFileName] = fileContent
+
+			}
+		}
+
+		// Client certs
+	} else if strings.Contains(fileName, clientCertDir+string(os.PathSeparator)) {
+		if strings.Contains(fileName, clientCertFile) {
+			clCertJSON, conversionErr := utills.ToJSON(fileContent)
+			if conversionErr != nil {
+				loggers.LoggerAPI.ErrorC(logging.ErrorDetails{
```

Comment:
we may include api details (api name, context, etc..) in the message

Reply:
Added API name and the version.